packages/core/src/App/Components/Routes/route-with-sub-routes.jsx

```diff
@@ -15,11 +15,14 @@ import { connect } from 'Stores/connect';
 
 const RouteWithSubRoutes = route => {
     const validateRoute = pathname => {
-        if (pathname.startsWith('/cashier') && !pathname.startsWith('/cashier/p2p/') && !!route.routes) {
+        if (pathname.startsWith('/cashier') && !pathname.includes('p2p') && !!route.routes) {
```

Comment:
using `startsWith` won't recognise this pathname: `cashier/p2p/` where it has an extra / at the end. its better to instead check if p2p is included in the pathname so we can handle p2p routing separately.